robots.txt

```diff
@@ -0,0 +1 @@
+Sitemap: http://example.com/sitemap_location.xml
```

Comment:
Should this be `https://shiro.apache.org/sitemap_location.xml` ?

Reply:
Thanks. I had it changed locally, but somehow it did not get commited.